app/src/main/java/org/schabi/newpipe/util/ShareUtils.java

```diff
@@ -194,17 +241,49 @@ private static String getDefaultAppPackageName(final Context context, final Inte
     /**
      * Open the android share menu to share the current url.
      *
-     * @param context the context to use
-     * @param subject the url subject, typically the title
-     * @param url     the url to share
+     * @param context         the context to use
+     * @param subject         the url subject, typically the title
+     * @param url             the url to share
+     * @param imagePreviewUrl the image of the subject
+     */
+    public static void shareText(final Context context,
+                                 final String subject,
+                                 final String url,
+                                 final String imagePreviewUrl) {
+        shareText(context, subject, url, imagePreviewUrl, true);
+    }
+
+    /**
+     * Open the android share sheet to share the current url.
+     *
+     * For Android 10+ users, a content preview is shown, which includes the title of the shared
+     * content.
+     * Support sharing the image of the content needs to done, if possible.
+     *
+     * @param context         the context to use
+     * @param subject         the url subject, typically the title
+     * @param url             the url to share
+     * @param imagePreviewUrl the image of the subject
+     * @param showPreviewText show the subject as an extra title of the Android share sheet if true
      */
-    public static void shareText(final Context context, final String subject, final String url) {
+    public static void shareText(final Context context,
+                                 final String subject,
+                                 final String url,
+                                 final String imagePreviewUrl,
+                                 final boolean showPreviewText) {
```

Comment:
Do we really need a switch to disable preview text? Why not just enable it always?

Reply:
I was thinking it was necessary for error reports, but we can show the translated string `Error report`.

Reply:
Without introducing more complexity I'd say `subject` should just be the translated `Error report`, what do you think?

Reply:
That's I want to say.

Reply:
Done, parameter removed.